test/topology/conftest.py

```diff
@@ -192,6 +193,12 @@ def fails_without_raft(request, check_pre_raft):
                                                          "experimental feature on"))
 
 
+@pytest.fixture(scope="function")
+def fails_aarch64(request, check_pre_raft):
```

Comment:
No need for this "check_pre_raft" thing, which I guess you copied from the fails_without_raft() fixture.

Reply:
Fixed in https://github.com/scylladb/scylladb/commit/ee1035acb12c9852578e07c6ca153b5e545c78dd